pkg/cli/alpha/generate.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alpha
+
+import (
+	"log"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kubebuilder/v3/pkg/rescaffold"
+)
+
+// NewScaffoldCommand return a new scaffold command
+func NewScaffoldCommand() *cobra.Command {
+	opts := rescaffold.MigrateOptions{}
+	scaffoldCmd := &cobra.Command{
+		Use:   "generate",
+		Short: "Re-scaffold an existing kuberbuilder project",
+		Long: `It's an experimental feature that has the purpose of
+		re-scaffolding the whole project from the scratch using the 
+		current version of KubeBuilder binary available.
+		# make sure the PROJECT file is in the 'input-dir' argument, the default is the current directory.
+		$ kubebuilder alpha generate --input-dir="./test" --output-dir="./my-output"
+		Then we will re-scaffold the project by Kubebuilder in the directory specified by 'output-dir'.
+		$ ls my-output/
+		Dockerfile  Makefile  PROJECT  README.md  cmd  config  go.mod  go.sum  hack
```

Comment:
```suggestion
```

I think we do not need to put an example of the output.

Reply:
Solved it and below. :)